cli/src/cli_builder.rs

```diff
@@ -269,13 +276,29 @@ impl<'a> NewCli<'a> {
         let mut node = &runner;
         let mut sm = &matches;
         while let Some((sub_name, sub_matches)) = sm.subcommand() {
-            node = node.children.get(sub_name).unwrap();
+            node = node
+                .children
+                .get(sub_name)
+                .ok_or_else(|| anyhow!("subcommand {sub_name} not found"))?;
             sm = sub_matches;
             if node.terminal {
                 break;
             }
         }
-        node.cmd.as_ref().unwrap().run_cmd(sm, &ctx).await
+
+        let Some(cmd) = node.cmd.as_ref() else {
+            bail!("action not found for subcommand");
+        };
+        let rt = if cmd.as_ref().type_id() == TypeId::of::<CustomCmd<CmdDiskImport>>() {
```

Comment:
I'd prefer that we make a new runtime in the disk upload subcommand or make threads manually as we deem appropriate

Reply:
I went with just using the `current_thread` executor everywhere based on our discussion.